### DISCUSSION TITLE
Rego Policy Comment Question

### DISCUSSION THREAD

**Discussion by Ronnie-personal** [open-policy-agent/community#423] [OPA and Rego] [answered]
Hi There,

I will ask a dumb question if you don't mind.

Does OPA ignore the comment statements when loading opa policy bundle to the agent? 
I would guess the comments (https://www.openpolicyagent.org/docs/latest/policy-language/#comments)  won't impact anything, including loading performance, memory consumption.

Could you kindly confirm?

Thanks.

**Comment by anderseknert** [comment, marked as answer]
Comments are always ignored :) the only impact I can think of is the time taken to realize something *is* a comment.

Metadata annotations would be the exception, as their content has meaning both inside and outside of policy evaluation.
